TheMET/TheMET/LoadingPlaceholderView/LoadingPlaceholderView.swift

```diff
@@ -43,6 +41,9 @@ class LoadingPlaceholderView: UIView {
             self.container.centerXAnchor.constraint(equalTo: self.centerXAnchor),
             self.container.centerYAnchor.constraint(equalTo: self.centerYAnchor),
             ])
+        NSLayoutConstraint.activate([
+            self.animationView.widthAnchor.constraint(equalTo: self.container.widthAnchor, multiplier: 0.5)
```

Comment:
Fixing the animtionView-width and fixing container's size will cause the textLabel to have a fixed size and cut the text if it is too long